common/tasks/fifo_scheduler.go

```diff
@@ -182,6 +182,10 @@ func (f *FIFOScheduler[T]) startWorkers(
 func (f *FIFOScheduler[T]) stopWorkers(
```

Comment:
Any reason why `FIFOScheduler` and `SequentialScheduler` didn't compose one another? They both look very identical.

Reply:
cc @yux0 I think SequentialScheduler is added after FIFO Scheduler for replication? Is there any special consideration in SequentialScheduler implementation we should be aware of if we want to consolidate the two impl?